src/Core/src/Platform/iOS/MauiWKWebView.cs

```diff
@@ -90,14 +90,23 @@ public void LoadHtml(string? html, string? baseUrl)
 				LoadHtmlString(html, baseUrl == null ? new NSUrl(NSBundle.MainBundle.BundlePath, true) : new NSUrl(baseUrl, true));
 		}
 
-		public void LoadUrl(string? url)
+		public async void LoadUrl(string? url)
```

Comment:
We need to add some wait to log errors. Maybe we can use the FireAndForget code that captures exceptions and logs them instead of crashing the app.

I think the implementations are:

```cs
private Task LoadUrlAsync(string url) { ... }
public void LoadUrl(string url) => LoadUrlAsync.FireAndForget();
```

Probably for net8 we can make ither the async public or use some loading events? Navigation failed or something?